end2end/playwright.config.ts

```diff
@@ -10,7 +10,7 @@ import { devices } from "@playwright/test";
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
-const config: PlaywrightTestConfig = {
+export default defineConfig({
```

Comment:
I used `defineConfig` because that was in [the snippet in the linked docs](https://playwright.dev/docs/test-configuration#basic-configuration).